src/iframe-helper.js

```diff
@@ -416,8 +416,12 @@ export class SubscriptionApi {
    * @param {boolean} is3p set to true if the iframe is 3p.
    * @param {function(!JsonObject, !Window, string)} requestCallback Callback
    *     invoked whenever a new window subscribes.
```

Comment:
@lannka Please see changes in this file. When we enter listenerForListener (line 199), event.origin = 'null'. (Not null, but the string 'null'). This affects the call to this.clientWindows_.push() on line 440 below. Later when we try to send to that subscriber, we can't send to 'null'. Hence my changes. Please let me know if you have any questions, or if there is a better way.